dashboard/src/model/WebPage.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+namespace dashboard\model;
+
+class WebPage
+{
+    private $gateways;
+
+    /**
+     * WebPage constructor.
+     * @param array $gateways
+     */
+    public function __construct(array $gateways)
+    {
+        $this->gateways = $gateways;
+        $this->render();
+    }
+
+    /**
+     * Generates the base page
+     */
+    public function render()
```

Comment:
Can this be private?

```suggestion
    private function render()
```